.circleci/config.yml

```diff
@@ -364,6 +371,9 @@ jobs:
             ./scripts/benchmark-runner.py compare \
                 --baseline_path ${BASELINE_OUTPUT_PATH} \
                 --target_path ${TARGET_OUTPUT_PATH} || true
+      - run-and-or-upload-benchmarks:
```

Comment:
following my comment above, it would be clearer if this could be just:

- upload-benchmark-results